agent/consul/config_endpoint.go

```diff
@@ -85,6 +85,7 @@ func (c *ConfigEntry) Get(args *structs.ConfigEntryQuery, reply *structs.ConfigE
 
 	// Create a dummy config entry to check the ACL permissions.
 	lookupEntry, err := structs.MakeConfigEntry(args.Kind, args.Name)
+	lookupEntry.GetEnterpriseMeta().Merge(&args.EnterpriseMeta)
```

Comment:
1. Is the merge working inplace, or does it have to be assigned?
2. That should happen after the error checking, no?!

Reply:
1. inplace
2. yes it definitely should. Good catch. I have moved it until after the err != nil check.

Reply:
👍 
since I looked it up, here is the link: https://github.com/hashicorp/consul/blob/6855a778c2cf4a4f72ce78fe2e7f3eeae29201e5/agent/structs/structs_oss.go#L25